sorted_keys.go

```diff
@@ -293,22 +293,30 @@ func MonotonicUnmarshalString(b []byte) (string, error) {
 
 /* Marshal Timestamp */
 func MonotonicMarshalTime(t time.Time) []byte {
-	bytes := MonotonicMarshalInt64(t.UnixNano())
+	bytes := make([]byte, 1+8+1+8)
+
 	bytes[0] = TimestampIdentifier
+	seconds := MonotonicMarshalInt64(t.Unix())
+	copy(bytes[1:9], seconds[1:9]) // Omit integer
+
+	bytes[9] = TimestampIdentifier
```

Comment:
The problem is, currently the marshalling/unmarshalling functions are designed so that they accept an array of n+1 bytes and ignore the first one.

The approach of just packing it into 17 bytes works, but really hurts your eyes when reading it. So I opted for this one, which loses a byte, but looks very pleasing.

Reply:
Oh you are right, forgot about that. Yeah that might not have been the best idea overall. But still for sure change the `1 + 8 + 1 + 8` to `2 * NumberMarshalLength`